plugins/analogix/fu-analogix-firmware.c

```diff
@@ -37,6 +37,7 @@ fu_analogix_firmware_parse (FuFirmware *firmware,
 	g_autoptr(GBytes) blob_srx = NULL;
 	g_autoptr(GBytes) blob_stx = NULL;
 
+	fu_ihex_firmware_set_padding_value (FU_IHEX_FIRMWARE(firmware), 0xFF);
```

Comment:
Doesn't this already get set in `fu_analogix_firmware_init()`? Or does that not work?

Reply:
Sorry, I did't see you have add it, thank you!